charts/premium/authelia/docs/Setup-Guide.md

```diff
@@ -18,97 +12,138 @@ LLDAP is a `Stable` train chart and therefore isn't supported at the same level
+      policy: one_factor
+      subject: "group:lldap_admin"
+```
 
 Please see [Authelia Rules](./authelia-rules) for more advanced rules.
 
-#### Setup Authelia Ingress
+### Setup Authelia Ingress
 
 - Make sure you're using the same domain as the `Default Redirection URL` above, so for me that's `auth.mydomain.com`
 
-![AutheliaIngress](./img/AutheliaIngress.png)
+```yaml
+// values.yaml
+ingress:
+  main:
+    enabled: true
+    integrations:
+      traefik:
+        enabled: true
+      certManager:
+        enabled: true
+        certificateIssuer: domain-0-le-prod
+    hosts:
+      - host: auth.example.com
+        paths:
+          - path: /
+            pathType: Prefix
+```
 
 ## Traefik ForwardAuth Setup
 
 - This part is straight forward as long as you have a working `Traefik` install, please see our [How-To](/charts/premium/traefik/how-to) if you need more info on getting that running.
 
-- Scroll down to `forwardAuth` and click `Add`
-
-![TraefikForwardAuth](./img/TraefikForwardAuth.png)
-
-- Name your `forwardauth` something you'll remember, since that's the middleware you'll add to your ingress going forward. Most people use `auth`
-- Address: `http://authelia.ix-authelia.svc.cluster.local:9091/api/verify?rd=https://auth.mydomain.com/` and replace the last part based on `mydomain.com`, and if you've changed ports/names you can get that from [`HeavyScript`](/
-- Check `trustForwardHeader`
-- Add the following `authResponseHeaders` (press `Add` 4 times)
-  - `Remote-User`
-  - `Remote-Groups`
-  - `Remote-Name`
-  - `Remote-Email`
+- The following is added to `Traefik` `values.yaml`,
+
+```yaml
+// values.yaml
+middlewares:
+  forwardAuth:
+    - name: auth
+      address: http://authelia.authelia.svc.cluster.local:9091/api/verify?rd=https://auth.example.com/
+      authResponseHeaders: 
+        - Remote-User
+        - Remote-Group
+        - Remote-Name
+        - Remote-Email
+      trustForwardHeader: true
+```
 
 ### Adding the forwardauth to your Apps
 
-The last step is adding the `forwardauth` along with the standard `ingress` settings for your app, for more info on setting ingress see the [ClusterIssuer Guide](/charts/premium/clusterissuer/how-to). What changes versus a standard setup is the `Traefik Middlewares` section, where you must add your `forwardauth` to the section.
-
-- In this example we use the same name as above, or `auth`. Click `Add` to the `Traefik Middlewares` section, and enter your `forwardauth` name.
-
-![TraefikForwardAuthMiddleware](./img/TraefikForwardAuthMiddleware.png)
+The last step is adding the `forwardauth` along with the standard `ingress` settings for your app, for more info on setting ingress see the [ClusterIssuer Guide](/charts/premium/clusterissuer/how-to).
+
+- In this example we use the same name as above, or `auth`.
+
+```yaml
+// values.yaml
+ingress:
+  main:
+    enabled: true
+    integrations:
+      traefik:
+        enabled: true
+        middlewares:
+          - name: auth
```

Comment:
I would add the namespace of traefik to the middleware to be safe.

Reply:
Shouldn't default work? Trying to KISS this as much as poissible.